rust/phoenix-channel/src/lib.rs

```diff
@@ -296,15 +363,9 @@ where
 
                         tracing::debug!(?backoff, max_elapsed_time = ?self.reconnect_backoff.max_elapsed_time, "Reconnecting to portal on transient client error: {e}");
 
-                        self.state = State::Connecting(Box::pin(async move {
-                            tokio::time::sleep(backoff).await;
```

Comment:
Where did the backoff sleep go?

Reply:
oops, lmk what you think now, introduced a helper function to avoid a bit of repetition